packages/fast-components-styles-msft/src/progress/index.ts

```diff
@@ -29,83 +29,85 @@ const styles: ComponentStyles<ProgressClassNameContract, DesignSystem> = (
         },
         progress_valueIndicator: {
             background: brandColor,
-            borderRadius: "2px",
+            borderRadius: "100px",
             height: "100%",
         },
         progress_indicator: {
             position: "relative",
             display: "flex",
             alignItems: "center",
             width: "100%",
+            overflow: "hidden",
+            borderRadius: "100px",
+            height: toPx(designSystem.designUnit),
+            background: backgroundColor,
+            maskImage: "-webkit-radial-gradient(white, black)",

```

Comment:
Interesting - did you verify in all browsers?

Reply:
The issue is that if a parent element has a `border-radius` property set and its `overflow` set to `hidden` Safari does not properly mask the child elements with the set corner radius... you just end up with a square clip. This is a fix for Safari only (Edge and Chrome seem to clip properly) and I verified it fixed the issue in a codepen sample on @janechu machine. If you don't mind, would be good for you to smoke this review and verify it fixes the issue in Safari (Since I'm not on a Mac)

Reply:
Tested on Safari - seems the same as Chrome.